reactivesocket-core/src/main/java/io/reactivesocket/internal/Responder.java

```diff
@@ -735,11 +735,11 @@ public void request(long n) {
                                     if (rn.intValue() > 0) {
                                         // initial requestN back to the requester (subtract 1
                                         // for the initial frame which was already sent)
-                                        child.onNext(Frame.RequestN.from(streamId, rn.intValue() - 1));
+                                        child.onNext(Frame.RequestN.from(streamId, Math.min(Integer.MAX_VALUE, rn.intValue() - 1)));
```

Comment:
Actually, based on the spec, the `request-n` is a 64bits value (it should actually be a 63bits, but that's another story).
https://github.com/ReactiveSocket/reactivesocket/blob/master/Protocol.md#request-n-frame

So instead of downcasting everything to int, we upcasting everything to long.
The `RequestNFrame` is bogus.